workers/typescript/src/omes.ts

```diff
@@ -0,0 +1,98 @@
+import { Command } from 'commander';
```

Comment:
nit: [arg](https://www.npmjs.com/package/arg) is much lighter and IMHO nicer to use.

Reply:
Looks like it doesn't support help strings, so I'm not pushing here.